packages/DatePicker/src/DatePicker.js

```diff
@@ -1,30 +1,24 @@
 import React from "react";
 import PropTypes from "prop-types";
-import moment from "moment";
 import momentPropTypes from "react-moment-proptypes";
 
-import CalendarIcon from "@paprika/icon/lib/Calendar";
-import Input from "@paprika/input";
 import Popover from "@paprika/popover";
-import useI18n from "@paprika/l10n/lib/useI18n";
+import DateInput from "@paprika/date-input";
 import useDebounce from "@paprika/helpers/lib/hooks/useDebounce";
-import usePrevious from "@paprika/helpers/lib/hooks/usePrevious";
 import isElementContainsFocus from "@paprika/helpers/lib/dom/isElementContainsFocus";
 
 import Calendar from "./components/Calendar";
-import DateInput from "./components/DateInput";
+import DateInputPropsContainer from "./components/DateInput";
 import DatePickerPopover from "./components/DatePickerPopover";
 import { extractChildrenProps } from "./helpers";
 
 import { calendarPopoverStyles } from "./DatePicker.styles";
 
-const INPUT_PARSE_ERROR = "INPUT_PARSE";
-
 const propTypes = {
   children: PropTypes.node,
 
   /** Date format used while entering and parsing user input. */
-  dataFormat: PropTypes.string,
+  dateFormat: PropTypes.string,
```

Comment:
We named it `dataFormat` because it's used when parsing the data from string to moment date, it's not a typo 😂 
Maybe people already discussed renaming it and I missed

Reply:
Oooohhh... Now I think I should revert changes related to renaming this prop. @allison-c @nahumzs WDYT? Should I?

Reply:
But it is very strange. I see comment just above this line: `/** `**Date format** `used while entering and parsing user input. */`. But prop itself called `dataFormat`.

Reply:
Let's ask @nahumzs 😂

Reply:
The description for `humanFormat` is similar `/** Date format used while displaying date. It should be human-friendly and spelled out, default is MMMM DD,YYYY */`

But I can see if people feel confusing, we probably can rename it..

Reply:
to me seems more natural `dateFormat` as Danil is proposing.

But also we don't need to break the api we could just deprecate the prop and add dateFormat, that way we don't break the API and previous components can still keep working without problem.

I found a propType function that could help us with this https://github.com/thefrontside/deprecated-prop-type/blob/master/deprecated.js

cc @mikrotron

Reply:
This PR is already a breaking change for date picker, so I think we can break the api maybe?

Reply:
I'm reluctant to maintain deprecated props for any of our components that are `< v1.0`.  We should probably start releasing some `v1.0` components, but if it's `0.x` then that suggests the API isn't quite stable, and we shouldn't worry too much about introducing breaking changes.

Reply:
@nahumzs I agree with Mike. I think we can introduce this breaking change without deprecation warnings.

-------

@allison-c 
> This PR is already a breaking change for date picker, so I think we can break the api maybe?

Actually, changing this prop name is the only one breaking change for DatePicker. There is no other breaking changes.

Reply:
ok I'm good with that.